documentation/vscode_and_bolt.md

```diff
@@ -0,0 +1,73 @@
+# Setting up VS Code for Bolt
+
+Bolt [configuration](configuring_bolt.md) and [inventory](inventory_file_v2.md)
+files are written in YAML that conforms to a particular specification. It also
+
+It can speed up the development cycle and reduce errors if your authoring
+environment knows about these specifications, because you can get real-time
+validation and data type checking as you write.
```

Comment:
```suggestion
Bolt configuration and inventory files are written in YAML that 
conforms to a particular specification. Making your authoring 
environment aware of these specifications gives you real-time 
validation and data type checking as you write, speeding
up the development cycle and reducing errors.
```
Wherever possible, lets favor offloading these kinds of links to "Related info" sections - especially in opening paragraphs. Sometimes lots of links can be distracting to the user. If we're linking out to the same docs in several places in the text, this is usually a good indication that we can cover those topics in the related info section.

Reply:
Good to know! I'll keep this in mind for future docs. :)